### PR TITLE
SSH: add some info and a call for it

### PR DESCRIPTION
**Pull request by sunishigh**
Adds info for two cases that I stumbled upon when using the docs.

### CODE REVIEW

File: src/gettingstarted/template/check-status.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Build, Deploy, Done!
 
-Once you have configured the template application in the previous step, Platform.sh will build your project for you.
+Once you have configured the template application in the previous step, Platform.sh will build your project for you. If you created a blank project, be sure to read on [setting up your SSH keys](/development/ssh.md) before trying to upload your files.
```
Comment (Crell) [inline src/gettingstarted/template/check-status.md]:
```suggestion
Once you have configured the template application in the previous step, Platform.sh will build your project for you. If you created a blank project, be sure to [set up your SSH keys](/development/ssh.md) before trying to upload your files.
```

File: src/development/ssh.md [hunk 2]
```diff
@@ -172,6 +181,12 @@ debug1: No more authentication methods to try.
 Permission denied (publickey).
 ```
 
+or
+
+```bash
+$ GIT_SSH_COMMAND="git -v" git clone [REPO-URL]
+```
```
Comment (Crell) [inline src/development/ssh.md]: :+1:

File: src/development/ssh.md [hunk 3]
```diff
@@ -137,6 +137,15 @@ $ ssh-add path-to-your-key
...
 
+### Try altering your SSH configuration
+
+You can try to append some directives to your `~/.ssh/config` file as in the example below, where `id_platformsh` is the public key file.
+
+    Host platform.sh
+    IdentityFile ~/.ssh/id_platformsh
+
+With this configuration, you should be asked for a password if your key has one.
+
```
Comment (Crell) [inline src/development/ssh.md]: I'm not sure about this addition.  "Altering" isn't a good description here of what's being proposed.  I'm also not sure why explicitly identifying the key would make a difference.  If it does, the reason for that should be explained.

Comment (sunishigh) [reply src/development/ssh.md]: Added explanation about OpenSSH (not mentioned explicitly) not looking at non-default key file names (based on man ssh).

**Comment by Crell** [comment]
Much improved, thanks!

**Pull request merged**